creator-node/src/services/prometheusMonitoring/prometheusRegistry.js

```diff
@@ -0,0 +1,43 @@
+const promClient = require('prom-client')
+
+module.exports = class PrometheusRegistry {
+  init({ collectDefaultMetrics = true }) {
+    this.register = new promClient.Registry()
+
+    // Add a default label which is added to all metrics
+    this.register.setDefaultLabels({
+      serviceType: 'audius_content_node'
+    })
+
+    this.setupDefaultMetrics(collectDefaultMetrics)
+  }
+
+  getAllMetricData() {
+    return this.register.metrics()
+  }
+
+  async getMetricInstance(name) {
+    return this.register.getSingleMetric(name)
+  }
+
+  setupDefaultMetrics(collectDefaultMetrics) {
+    if (collectDefaultMetrics) {
+      promClient.collectDefaultMetrics({
+        register: this.register
+      })
+    }
+
+    this._createHTTPRequestTimerMetric()
+  }
+
+  _createHTTPRequestTimerMetric() {
+    const timer = new promClient.Histogram({
+      name: 'http_request_duration_seconds',
+      help: 'http_request_duration_seconds',
```

Comment:
This should be a human readable description of what the `name` field holds.

Reply:
yep will do